test/language/expressions/this/S11.1.1_A3.1.js

```diff
@@ -17,7 +17,7 @@ if (MyFunction() !== this) {
 }
 
 //CHECK#2
-function MyFunction() {return eval("this")}
-if (MyFunction() !== this) {
-  $ERROR('#2: function MyFunction() {return eval("this")} MyFunction() === this. Actual: ' + (MyFunction()));
+function MyFunction2() {return eval("this")}
```

Comment:
it's not a standard at all, these are just all very old legacy tests, dated from 2009.

I just went bananas in these tests because there are some many things I want to fix, I went short on naming.